templates/part.calendarlist.item.php

```diff
@@ -205,4 +206,28 @@ class="app-navigation-entry-menu hidden">
 			</span>
 		</li>
 	</ul>
+	<div class="publishing" ng-if="item.calendar.isPublishable()">
+		<input type="checkbox" name="publish"
+			   id="checkbox_publish_calendar_{{ $index }}"
+			   ng-model="item.calendar.published" value="edit"
+			   ng-change="togglePublish(item)">
+		<label for="checkbox_publish_calendar_{{ $index }}"> <?php p($l->t('Share link')); ?></label>
```

Comment:
please use the fancy checkbox styles from the server :) 
see https://github.com/nextcloud/calendar/pull/79